src/api/deployment.ts

```diff
@@ -10,6 +10,31 @@ import {
 } from '../types';
 import { callAPI } from './skygear';
 
+export async function validateDeployment(
+  context: CLIContext,
+  deployments: { [name: string]: DeploymentItemConfig },
+  hooks: HookConfig[]
+): Promise<void> {
+  const deploymentsRequestPayload: {
+    [key: string]: ReturnType<
```

Comment:
Is it intentional that type `DeploymentItemConfigPayload` is not used?

Reply:
The reason I can think of is that `validateDeployment` is copied from `createDeployment` and `createDeployment` is older than `DeploymentItemConfigPayload`. I will update both to use `DeploymentItemConfigPayload`. Good catch!